docs/framework/winforms/advanced/international-fonts-in-windows-forms-and-controls.md

```diff
@@ -80,5 +80,4 @@ textBox1.Font = new System.Drawing.Font(this.Font, textBox1.Font.Style);
   
 ## See also
 
-- [Globalizing Windows Forms applications](globalizing-windows-forms.md)
```

Comment:
@rpetrusha I found this one and I removed it.

Reply:
Thanks, @Thraka.